shared/actions/teams.js

```
@@ -323,6 +323,27 @@ const addToTeam = (_, action) => {
     })
 }
 
+const reAddToTeam = (state, action) => {
+  const {teamname, username} = action.payload
+  return RPCTypes.teamsTeamReAddMemberAfterResetRpcPromise(
+    {
+      id: state.teams.teamNameToID.get(teamname, ''),
```

Can you get this before the call and log/throw if it's not found?